offchain-modules/packages/app-multisign-server/src/ckbSigner.ts

```diff
@@ -354,6 +367,12 @@ export async function signCkbTx(params: collectSignaturesParams): Promise<SigRes
         return new SigResponse(err);
       }
       break;
+    case 'unlock':
```

Comment:
`unlock tx message to sign` is different from `mint tx message to sign`, can not use the same code https://github.com/nervosnetwork/force-bridge/blob/09bb874970dfabe4f36a46cad4e5b0ad4ccfe02d/offchain-modules/packages/app-multisign-server/src/ckbSigner.ts#L361

Reply:
fixed: switch if sigType is mint or unlock use different code to get the message that to sign